<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>264</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="VideoBTN">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Video</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="PortocaliuBTN">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Portocaliu</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="GalbenBTN">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Galben</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="VerdeBTN">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Verde</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="AlbastruBTN">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Albastru</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="VioletBTN">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Violet</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="RosuBTN">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Rosu</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>393</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
